storage/tianmu/common/data_format.cpp

```diff
@@ -27,26 +27,26 @@
 namespace Tianmu {
 namespace common {
 
-int DataFormat::no_formats = 0;
+int DataFormat::no_formats_ = 0;
 
-std::map<std::string, DataFormatPtr> DataFormat::df_map = {{"EDF::TRI_UNKNOWN", std::make_shared<TxtDataFormat>()}};
+std::map<std::string, DataFormatPtr> DataFormat::df_map_ = {{"EDF::TRI_UNKNOWN", std::make_shared<TxtDataFormat>()}};
 
 DataFormatPtr DataFormat::GetDataFormat(const std::string &name) {
-  auto it = df_map.find(boost::trim_copy(boost::to_upper_copy(name)));
-  return it != df_map.end() ? it->second : DataFormatPtr();
+  auto it = df_map_.find(boost::trim_copy(boost::to_upper_copy(name)));
```

Comment:
`df_map` is a map for what? the readers can understand the basic idea of the code via the literal words.